src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -546,7 +546,7 @@
     <Compile Include="$(CommonPath)System\Threading\Tasks\TaskCompletionSourceWithCancellation.cs"
              Link="Common\System\Threading\Tasks\TaskCompletionSourceWithCancellation.cs" />
   </ItemGroup>
-  <ItemGroup Condition="('$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true') and '$(TargetsBrowser)' != 'true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' and '$(TargetsBrowser)' != 'true'">
```

Comment:
`and '$(TargetsBrowser)' != 'true'" ` is redundant here, right?

It'd be nice to clean up other csproj files which have this extra logic

Reply:
ah Yes, we changed that. i will take a look and clean this up